tests/test_model.py

```diff
@@ -308,10 +303,6 @@ def test_sort(self) -> None:
 
 class TestModelIdentifiableAction(TestCase):
 
-    def test_no_params(self) -> None:
```

Comment:
instead of removing the test, i would rather refactor it - test that no exception is risen.
What do you think?